src/assignments/tech-jobs-oo.rst

```diff
@@ -1,6 +1,586 @@
 .. _tech-jobs-oo:
 
-Assignment #3: Tech Jobs (Object-Oriented Edition)
-===================================================
+Assignment #2: Tech Jobs (Object-Oriented Edition)
+===========================================================
 
-Noting here yet!  Check back soon.
+Introduction
+------------
+
+Your apprenticeship at LaunchCode is going well! Only a few weeks in and you’re
+regularly making contributions to code that will eventually be used by all
+LaunchCode staff.
+
+Your last task was to get the prototype Tech Jobs app in good shape. Now it’s
+time to advance the underlying structure of the program.
+
+Your mentor on this project is Sally, one of the developers at LaunchCode. She
+regularly supports coders who are just getting started with their careers.
+
+.. figure:: figures/LC-Sally.png
+   :scale: 50%
+   :alt: Sally's LaunchCode avatar.
+
+After seeing your strong work with your last project, Blake reported that you
+performed well and learned quickly. Because of your success, he and Sally feel
+comfortable assigning you to a set of tasks that are a notch up in difficulty.
+
+Sally completed some initial work on the project and left you some TODOs.
+
+Learning Objectives
+--------------------
+
+In this project, you’ll show that you can:
+
+#. Read and understand code written by others.
+#. Work with *objects* to encapsulate data and methods.
+#. Use the generator in IntelliJ to automate routine tasks.
+#. Use unit testing and Test-Driven-Development (TDD) to verify and create new
+   methods.
+#. Apply the concept of inheritance to streamline your classes (the DRY
+   idea---Don't Repeat Yourself).
+
+Get the Starter Code
+---------------------
+
+#. Set up a local copy of the project. Visit the
+   `repository page <https://github.com/LaunchCodeEducation/java-web-dev-techjobs-oo.git>`__
```

Comment:
The main package for the assignment is `org.launchcode.TechJobsOO`, which goes against [Java naming conventions](https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html) (package names should be all lowercase).

Reply:
Well, phOOey.

Fixed in GitHub and GitLab now.